backend/mora/service/employee.py

```diff
@@ -317,7 +345,18 @@ def get_employee(id):
     '''
     c = common.get_connector()
 
-    r = get_one_employee(c, id, full=True)
+    details = flask.request.args.get("employeedetails", EmployeeDetails.FULL)
```

Comment:
The documentation should mention this parameter. The fact that you're acting on an employee is implied by the endpoint; hence including the string `employee` in the argument name is redundant. Furthermore, if it must be optional, I'd prefer a dedicated boolean parameter, such as `integregationdata=1` — see `utils.get_args_flag()`.

What does generating this data involve? If next to nothing, we might as well include this by default and drop the parameter, as @agger-magenta suggests.

Reply:
:+1: